scripts/api/data/metadatablocks/citation.tsv

```diff
@@ -12,7 +12,7 @@
 	authorName	Name	The author's Family Name, Given Name or the name of the organization responsible for this Dataset.	FamilyName, GivenName or Organization	text	8	#VALUE	TRUE	FALSE	FALSE	TRUE	TRUE	TRUE	author	citation
 	authorAffiliation	Affiliation	The organization with which the author is affiliated.		text	9	(#VALUE)	TRUE	FALSE	FALSE	TRUE	TRUE	FALSE	author	citation
 	authorIdentifierScheme	Identifier Scheme	Name of the identifier scheme (ORCID, ISNI).		text	10	- #VALUE:	FALSE	TRUE	FALSE	FALSE	TRUE	FALSE	author	citation	http://purl.org/spar/datacite/AgentIdentifierScheme
-	authorIdentifier	Identifier	Uniquely identifies an individual author or organization, according to various schemes.		text	11	#VALUE	FALSE	FALSE	FALSE	FALSE	TRUE	FALSE	author	citation	http://purl.org/spar/datacite/AgentIdentifier
+	authorIdentifier	Identifier	Uniquely identifies an individual author or organization, according to various schemes.		textbox	11	##authorIdentifierScheme	FALSE	FALSE	FALSE	FALSE	TRUE	FALSE	author	citation	http://purl.org/spar/datacite/AgentIdentifier
```

Comment:
Is the double `#` character here a typo? Do not see any other instance of it in this file.

Reply:
It is not a typo, in this comment by @scolapasta https://github.com/IQSS/dataverse/issues/1625#issuecomment-567567073 recommended using the tag like that.

Reply:
Ah, sorry, @mderuijter. Did not notice that recommendation. Resolving this comment.